A featured organ can search with several facetIds

ignapas opened the pull request:
# Description

For adding the featured data "Nerves", the corresponding button in the Home Page needs to link to a search with several facetIds selected.


## Type of change

Delete those that don't apply.

- New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Manual test


# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

egauzens reviewed (approved):
nice, this is exactly what I meant. Now we can add a field to contentful for other organ keywords that a featured data icon should be paying attention to (in this case it sounds like ganglia at least)

ignapas commented:
I added the capability to use a new property in Contentful called ``containsSearch`` to be able to match other organs with the keywords contained by this property. The property is also added in Contentful. I will replace Pancreas with Nerve once this is merge. There is also the other PR about the resource types becoming an array. And as reminder, I don't have privileges to merge 👍🏼
